edxval/api.py

```diff
@@ -18,6 +17,7 @@
 from lxml.etree import Element, SubElement
 from pysrt.srtexc import Error
 
+import cachetools.func
```

Comment:
this import should be in 2nd level as this is a 3rd party package. Not sure why quality did not fail on this. You can try isort to fix this.

Reply:
I've already used isort on this file